### ISSUE TITLE
Labeling tool

### ISSUE THREAD

**Issue by MonchoG** [ailabstw/Label360#1]
Dear Developer,
I read the paper - very nice work, congratulations ! My question for you is how can I use your tool to label my own dataset ? I have some 360 footage of sewer pipes, which I want to label for training neural networks. Is that possible with your tool ?
